nautobot_circuit_maintenance/templates/nautobot_circuit_maintenance/circuit_maintenance_overview.html

```diff
@@ -0,0 +1,129 @@
+                        </tr>
+                    </thead>
+                    <tbody>
+                        {% for maintenance in upcoming_maintenances %}
+                            {% for circuit in maintenance.circuits %}
+                                {% if circuit.termination_a.connected_endpoint %}
+                                    <tr>
+                                        <td>
+                                            {% with device_obj=circuit.termination_a.connected_endpoint.parent %}
+                                            <a href="{{ device_obj.get_absolute_url }}">{{ device_obj.name }}</a> 
+                                            {% endwith %}
+                                        </td>
+                                        <td>
+                                            {{ maintenance.start_time | date:'Y-m-d H:i e' }}
+                                        </td>
+                                    </tr>
+                                {% endif %}
+                                {% if circuit.termination_z.connected_endpoint %}
+                                    <tr>
+                                        <td>
+                                            {% with device_obj=circuit.termination_z.connected_endpoint.parent %}
+                                            <a href="{{ device_obj.get_absolute_url }}">{{ device_obj.name }}</a>
+                                            {% endwith %}
+                                        </td>
+                                        <td>
+                                            {{ maintenance.start_time | date:'Y-m-d H:i e' }}
+                                        </td>
+                                    </tr>
+                                {% endif %}
+                            {% endfor %}
+                        {% endfor %}
+                    </tbody>
+                </table>
+            </div>
+        </div>
+        <div class="card">
+            <div class="container">
+                <h3 class="text-left">Metrics</h3>
+                <table class="table table-hover table-headings">
+                    <thead>
+                        <tr>
+                            <th>Metric</th>
+                            <th>Metric Value</th>
+                        </tr>
+                    </thead>
+                    <tbody>
+                        {% for cm_metric_key, cm_metric_value in circuit_maint_metric_data.items %}
+                        <tr>
+                            <td>{{ cm_metric_key }}</td>
+                            <td>{{ cm_metric_value }}</td>
+                        </tr>
+                        {% endfor %}
+                    </tbody>
+                </table>
+            </div>
+        </div>
+    </div>
+</div>
+
+{% table_config_form table table_name="ObjectTable" %}
+{% endblock %}
+{% block javascript %}
+<script src="{% static 'js/tableconfig.js' %}"></script>
```

Comment:
@jvanderaa , could you give explain (point to doc) what are we using this for?

Reply:
@itdependsnetworks I took this from the GC plugin. Any insight?

Reply:
Removed it for now. It is used for table selection.

Reply:
Let's get this merged and released?

Reply:
It's used for customizing a table view to specify which columns are displayed. If you're removing it, you should also remove the `table_config_form` template tag as well.